modules/learn/pages/data/durability.adoc

```diff
@@ -58,258 +62,303 @@ The correspondences are as follows:
-Active vBuckets thereby lost are replaced by the promotion of replica vBuckets, on the surviving nodes.
-The maximum number of sequential automatic failovers is configurable by the administrator.
+xref:learn:clusters-and-availability/automatic-failover.adoc[Automatic Failover] removes a non-responsive node from the cluster after a configurable timeout.
+When Couchbase Server fails over a node, it promotes replica vBuckets on surviving nodes to replace the lost active vBuckets.
+You can configure the maximum number of sequential automatic failovers.
 
-In cases where commitment based on _persistToMajority_ has occurred, but no further propagation of the new data across the cluster has yet occurred, automatic failover of the nodes containing the new data results in the data's loss &#8212; since no updated replica vBucket yet exists elsewhere on the cluster.
+Automatic failovers can cause the loss of a durable write.
+If you perform a successful durable write using `persistToMajority` but the data has not yet propagated in the cluster beyond the majority, automatic failover of the majority nodes causes data loss.
+This loss happens because there is no updated replica vBucket in the cluster.
 
-For example, if a bucket has two replicas, the total number of nodes on which the data resides is _three_; and the _majority_ of nodes, on which persistence must occur prior to commitment, is _two_.
-After commitment, if those two nodes become unresponsive, automatic failover, if configured to occur up to a maximum of two times, allows those two nodes to be failed over _before_ the durable write has been made persistent on the third node.
-In such a case, the durable write is lost, and the success message already delivered to the application rendered false.
+For example, suppose you configure a bucket with two replicas.
+Also, you allow up to two automatic failovers.
+The data resides on three nodes and the majority for persistence is two nodes.
+If the two nodes making up the majority for a durable write become unresponsive after the write commits, Couchbase Server fails over both nodes.
+The failover can happen before the durable write has persisted on the third node.
+In this case, Couchbase Server loses the durable write, and the application receives a false success message.
 
-To prevent this, and thereby maintain guaranteed protection, at least one of the unresponsive nodes containing the new data should _not_ be failed over; and auto-failover should be allowed to occur sequentially only up to the number of times that supports this requirement.
-This may be best achieved as described immediately below, in xref:learn:data/durability.adoc#preserving-durable-writes[Preserving Durable Writes].
+To prevent this potential loss of data, do not allow automatic failover of all nodes that contain the new data.
+Set the maximum number of sequential automatic failovers to a value that protects at least one node with the new data.
```

Comment:
The following section explains something else, the AFO setting that prevents loss of a durable write/majority.

Something close to that can be accomplished by choosing a specific AFO max count, but that comes with other downsides when you have services other than KV in the cluster.

This should be pitched as two options, and I think that we should promote the specific AFO setting for this over use of the max count in this way as it's more user friendly, if more restrictive in some scenarios.